include/experimental/__p0009_bits/submdspan.hpp

```diff
@@ -397,7 +397,8 @@ constexpr auto _submdspan_impl(
   auto map = _handled.make_layout_mapping(src.mapping());
   auto acc_pol = typename AP::offset_policy(src.accessor());
   return mdspan<
-    ET, decltype(map.extents()), typename decltype(_handled)::layout_type, decltype(acc_pol)
+    ET, remove_const_t<remove_reference_t<decltype(map.extents())>>,
+        typename decltype(_handled)::layout_type, remove_const_t<remove_reference_t<decltype(acc_pol)>>
```

Comment:
Why is it not `remove_cv_t`?

Reply:
we don't have volatile on this thing or?